setup.py

```diff
@@ -80,7 +80,13 @@
     long_description_content_type="text/markdown",
     author="ApeWorX Ltd.",
     author_email="admin@apeworx.io",
-    url="https://github.com/ApeWorX/ape",
+    url="https://apeworx.io",
+    project_urls={
+        "Documentation": "https://docs.apeworx.io/ape/",
+        "Funding": "https://gitcoin.co/grants/5958/ape-maintenance-fund",
+        "Source": "https://github.com/ApeWorX/ape",
+        "Tracker": "https://github.com/ApeWorX/ape/issues",
```

Comment:
If you add `Twitter` here, I think it will show up with the little bird! Wondering if there are any more?

Reply:
Twitter is a good one!

Reply:
```suggestion
        "Tracker": "https://github.com/ApeWorX/ape/issues",
        "Twitter": "https://twitter.com/ApeFramework",
```